docs/2.3/saml.md

```diff
@@ -297,3 +297,173 @@ $ tctl saml export adfs
 Save the output to a file named `saml.crt`, return back to ADFS, open the
 "Relying Party Trust" and add this file as one of the signature verification
 certificates.
+
+## One Login
+
+This guide will cover how to configure Teleport to authenticate users via SAML
+using [One Login](https://www.onelogin.com/) as a SAML provider.
+
+### Enable SAML Authentication
+
+First, configure Teleport auth server to use SAML authentication instead of the local
+user database. Update `/etc/teleport.yaml` as show below and restart the
+teleport daemon.
+
+```bash
+...
+auth_service:
+    # Turns 'auth' role on. Default is 'yes'
+    enabled: yes
+
+    # defines the types and second factors the auth server supports
+    authentication:
+        type: saml
+...
+```
+
+### Confiugre One Login Application
+
+First, create a SAML 2.0 Web App in SAML configuration section:
+
+![Create APP](img/onelogin-saml-1.png?raw=true)
+
+!!! tip "Important":
+    Make sure to pick `SAML Test Connector (SP)` and not `SAML Test Connector (IdP)`,
+    because teleport only supports `SP` - service provider initiated SAML flows.
+
+**Configure the App**
+
+Set `Audience`, `Recipient`, `ACS (Consumer) URL Validator` and to the same value:
+
+`https://teleport.example.com/v1/webapi/saml/acs` where `teleport.example.com` is the
+public name of the teleport web proxy service:
+
+![Configure APP](img/onelogin-saml-2.png)
+
+Teleport needs to assign groups to users. Configure the application with some parameters
+exposed as SAML attribute statements:
+
+![Configure APP](img/onelogin-saml-3.png)
+![Configure APP](img/onelogin-saml-4.png)
+
+!!! tip "Important":
+Make sure to check `Include in SAML assertion` checkbox.
+
+Add users to the application:
+
+![Configure APP](img/onelogin-saml-5.png)
+
+### Create a Teleport SAML Connector
+
+Now, create a SAML connector [resource](admin-guide#resources).
+Write down this template as `onelogin-connector.yaml`:
+
+```bash
+kind: saml
+version: v2
+metadata:
+  name: OneLogin
+  namespace: default
+spec:
+  acs: https://teleport.example.com/v1/webapi/saml/acs
+  attributes_to_roles:
+    - {name: "groups", value: "admin", roles: ["admin"]}
+    - {name: "groups", value: "dev", roles: ["dev"]}
+  display: Login with One Login
```

Comment:
I think if you have "Login with" in the display param the UI ends up reading "Login with Login with".

Also their official name is OneLogin without spaces